packages/material-ui/src/ListItemIcon/ListItemIcon.js

```diff
@@ -9,6 +9,7 @@ export const styles = theme => ({
     marginRight: 16,
     color: theme.palette.action.active,
     flexShrink: 0,
+    display: 'flex',
```

Comment:
It was `block` before technically. Why is this necessary now?

Reply:
There was an issue with [`display: inline-block`](https://stackoverflow.com/questions/7642883/why-arent-these-elements-with-displayinline-block-properly-aligned) in the child `svg` element adding extra vertical space to the `ListItemIcon` wrapper. This can be reproduced by replacing an icon with a `div` element of the same dimensions as the `svg`, with `display: inline-block` as such:

```
<div class="MuiListItemIcon-root-XXX"> // Set display: block
  <div class="mock-div"></div> // display: inline-block, dimensions 24px by 24px
</div>
```
For some reason this adds extra vertical space inside the `ListItemIcon`, adding an extra 4px to its height and causing the entire list item to look awkward (hence the failed Argos run).

Setting the display to `flex` seems to resolve this weird quirk in how `inline-block` elements behave, maybe due to the way `flex` resizes its children to fit?

Another solution for this is (as mentioned in the link above) to add `vertical-align: top` to the styling of the list item icon children, but I figured that the way I chose is more concise and affects fewer components.

Reply:
Weird. Hope this doesn't cause any issues. 

@oliviertassinari Maybe add a warning to the release notes that the style of `ListItemIcon` has slightly changed to fix the broken behavior with `color`?

Reply:
Yeah, it might lead to different behavior depending on the situations.

Reply:
Setting the display to `inline-flex` also resolves the height issue, and might be more similar to the previous implementation (`inline-block`)

Reply:
@izyb Yeah, I agree, it's more transparent this way.